pyuvdata/conftest.py

```diff
@@ -3,26 +3,10 @@
 # Licensed under the 2-clause BSD License
 
 """Testing environment setup and teardown for pytest."""
-import os
-
 import pytest
 from astropy.utils import iers
 from astropy.time import Time
 from astropy.coordinates import EarthLocation
-import numpy as np
```

Comment:
I moved the test fixtures to object-specific files for better organization. It probably should have been a separate PR, it happened because I was cleaning up UVCal test fixtures and found the existing organization annoying.